ecosystem/typescript/sdk_v2/tests/unit/deserializer.test.ts

```diff
@@ -130,4 +129,187 @@ describe("BCS Deserializer", () => {
       deserializer.deserializeStr();
     }).toThrow("Reached to the end of buffer");
   });
+
+  it("deserializes a single deserializable class", () => {
```

Comment:
this flow feels more of an example rather than testing the serializer/deserializer classes? wdyt?